src/diff.sh

```diff
@@ -50,7 +50,7 @@ function diff_manager()
     return 0
   fi
 
-  if [[ -d $target_1 && -d $target_2 ]]; then
+  if [[ -d "$target_1" && -d "$target_2" ]]; then
```

Comment:
Don't add commits fixing problems that were introduced in this PR, you should squash this commit with the previous one. You should only ever create extra commits to fix things that have already been merged to the main branch as you don't want to alter the history, but on PRs you should definetly adapt the commits as to make them "perfect".